CRISPDM notebook.ipynb

```diff
@@ -98,10 +108,10 @@
    "outputs": [],
    "source": [
     "# load the data\n",
-    "diet = pd.read_csv(\"diet.csv\")\n",
-    "recipes = pd.read_csv(\"recipes.csv\")\n",
-    "requests = pd.read_csv(\"requests.csv\")\n",
-    "reviews = pd.read_csv(\"reviews.csv\")"
+    "diet = pd.read_csv('https://github.com/lauracorssac/AC-Harpy-Eagle/blob/92c6ec987957067ee9f0e69c74fe5d6bc84ae7d3/diet.csv?raw=true')\n",
```

Comment:
amiga pra mim ta funcionando normal o read_csv. Vamos olhar junto quem sabe? Acho o jeito de olhar do github nao muito bom. Nao vai funcionar quando nao tivermos com internet, por exemplo. Alem disso deve deixar mais lento.

Reply:
o jeito que eu to acessando o repositorio precisa de internet por si, então p mim não seria um problema
mas né, mudaria a celula p td mundo mesmo